.vscode/settings.json

```diff
@@ -22,5 +22,6 @@
 	},
 	"[scss]": {
 		"editor.rulers": [80]
-	}
+	},
+	"GitHooks.hooksDirectory": "c:\\Users\\User1\\OneDrive\\Documents\\GitHub\\Paradise\\.git\\hooks"
```

Comment:
Remove this change please, then this is good to go.

Reply:
and removed